app/javascript/components/UserProfileMenu/Menus.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import * as R from 'ramda'
+import { Link } from 'react-router'
+import {
+  Item,
+  Separator,
+  PreviousItem,
+  HeaderItem,
+  NextItem,
+} from '@zendeskgarden/react-dropdowns'
+import TranslationIcon from '@zendeskgarden/svg-icons/src/12/translation-exists-fill.svg'
+
+export const DefaultOfficeMenu = ({ offices, previousMenuValue, selectedItem, t }) =>
```

Comment:
Typically each file would contain one component, I would have expected three different files here.

Reply:
Okay I will do that, I was lazy 😛